packages/discussions/migrations/sqls/20201218010032-user-suspensions-up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE "discussionSuspensions" (
```

Comment:
do we also want to log who suspended that person?

Reply:
Funny, I thought so, too!

Now that we are two, I shall do so.